### DISCUSSION TITLE
SITATEJİ

### DISCUSSION THREAD

**Discussion by Mert5361** [IBAX-io/go-ibax#868] [Soru Cevap - Türkçe]
Arkadaşlar bir çok iyi projenin bu yıl içerisinde yanlış pazarlama ile ve yanlış adımlar atarak battığına şahit oldum ibcx eğer bunu başarabilirse güzel yerlere gelebilir ama bunun için yol haritasında yapması gereken herşeyi doğru tarihinde yapması gerek

**Comment by onurcux** [comment]
her kelimesinin altına imzamı atıyorum

**Comment by cryptopus58** [comment]
Çok doğru

**Reply by Yusufknk** [reply]
Sadece bekleyin ve gelişmeleri takip edin.
